.github/dependabot.yml

```diff
@@ -8,19 +8,7 @@ updates:
     allow:
       - dependency-type: "all"
     open-pull-requests-limit: 100
-  - package-ecosystem: "cargo"
-    directory: "/tests/host_tools/uffd/"
-    schedule:
-      interval: "weekly"
-      day: "monday"
-    allow:
-      - dependency-type: "all"
-    open-pull-requests-limit: 100
-  - package-ecosystem: "cargo"
-    directory: "/tests/integration_tests/security/demo_seccomp/"
-    schedule:
-      interval: "weekly"
-      day: "monday"
-    allow:
-      - dependency-type: "all"
-    open-pull-requests-limit: 100
+    groups:
+      firecracker:
+        patterns:
+          - "*"
```

Comment:
nit: can add a new line at the end.

Reply:
just out of curiosity, because I see requests like this a lot all over open source projects: Why?

Reply:
I think it is an unwritten law. Everyone believes that there are tools out there that don't handle the absences properly and it is always safer to add them.